explorer/config.js

```diff
@@ -0,0 +1,23 @@
+var fs = require('fs');
+
+var config = {};
+
+try {
+    var network = process.env.NETWORK || 'testnet';
+    var configFilename = __dirname + '/config.' + network + '.json';
+    var configContents = fs.readFileSync(configFilename);
+    config = JSON.parse(configContents);
+    console.log(configFilename + ' found.');
```

Comment:
@acroca still want me to delete this?

was there and it's useful to see what config was picked up

Reply:
btw, I like to comment my own PRs

Reply:
You can keep it if you think it's useful :)